scripts/release/mule/test/tests/post/verify_package_binaries.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# shellcheck disable=2116
+
+set -e
+
+echo "[$0] Verifying installed binaries..."
+
+RET=0
+RPMTMP=$(mktemp -d)
+
+cd "./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
+
+if [ "$PKG_TYPE" = deb ]
+then
+    dpkg -L algorand > "$RPMTMP/algorand.install"
+    dpkg -L algorand-devtools > "$RPMTMP/algorand-devtools.install"
+else
+    rpm -ql algorand > "$RPMTMP/algorand.install"
+    rpm -ql algorand-devtools > "$RPMTMP/algorand-devtools.install"
+fi
+
+ALGORAND_BINS=(
```

Comment:
This is okay, but I'd prefer if we could dynamically fetch this list from the script doing the packaging so we don't have to update in multiple places. Maybe over-optimizing, though.

Reply:
Yes, I called this out during the review, I agree it is a brittle area and should be hardened.  My preference is to commit as-is and come up with a design later.